client/src/components/Jobs/Jobs.scss

```diff
@@ -1,61 +1,73 @@
-.jobs-view {
-  /* icon */
-  th:nth-child(1),
-  td:nth-child(1) {
-    min-width: 2em;
+/* Dashboard Table */
+@media only screen and (min-width: 1670px) {
+  .jobs-view {
+    /* icon */    tr > *:nth-child(1) { width: 5%; }
+    /* name */    tr > *:nth-child(2) { width: 35%; }
```

Comment:
I think this needs a higher width to display the Session is Running button
![Screen Shot 2021-05-12 at 11 36 16 AM](https://user-images.githubusercontent.com/20326896/118012337-8c558780-b316-11eb-8767-2978201ad3e9.png)

Reply:
This is **not** immediately easy to fix by just tweaking table columns.

### Problem
If columns are made wider, then columns are too wide for tables that do __not__ have a job status button. I could conditionally tweak table columns, but **only** if the _whole_ table knows whether _any_ cells have job status button.

### Solutions

- __A. Tweak Table Col Widths If Table Has Buttons.__
    - If someone can add that knowledge to the table,
    - Then I can tweak column widths depending on whether any cell has job status button.
- __B. Automatically Tweak Col Widths For Buttons.__
    - Requires I remove `table-layout: fixed`.
    - I have considered, rejected, and reconsidered this multiple times.
- __C. Ask Design If They Want Different Solution.__
    - Tweaking column widths leads to A or B.
    - Anything else is probably a new ticket.

Reply:
**Reminder to Self**
Wes, don't fight this. This is not the only table column width bug you've met and will not be the last. Removing flex table cells has made `table-layout: fixed` effective which:
- limits creative column width solutions (like `min-width`)
- limits automatic column width adjustment (see "new maintenance")
- creates new bugs
- creates new maintenance

Reply:
**Done.** 107d046 There are now careful column widths for all breakpoints for both Job History and Dashboard "My Recent Jobs" to make sure icon nor status is cut off (and date is at least grok-able).